### PR TITLE
Update and make more user friendly to README.md

### PR DESCRIPTION
**Pull request by ayush-pandey047**
I have make README.md more attractive and added emoji. The plus point is that I added link on data.Brno, as it will help user to know about data.Brno if they don't know about it. I also added more content in it. 

if you like my work so accept this because it will motivate.(Hacktoberfest-accepted)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,41 @@
...
-- Filip Bučko
-- Tomáš Bártů
-- Šimon Šmída
+## 👥 Team Members
+- **David Chocholatý**
+- **Filip Bučko**
+- **Tomáš Bártů**
+- **Šimon Šmída**
+
+## 🛠️ Project Highlights
```
Comment (davidchocholaty) [inline README.md:25]: The `Project Highlights` part is IMHO not needed. Please remove this part.

File: README.md [hunk 2]
```diff
@@ -1,15 +1,41 @@
...
+- **Challenge Addressed:** 🌍 [Briefly describe the challenge or problem you aimed to solve.]
+- **Technologies Used:** 💻 [List the main technologies and tools you utilized.]
+- **Key Features:** ✨ [Highlight some notable features of your solution.]
+
+## 📱 App Preview
```
Comment (davidchocholaty) [inline README.md:30]: The same for the `App Preview`. Please remove this part too.

**Review by davidchocholaty — CHANGES_REQUESTED** [review]
Thank you very much for contributing to this repository and for your suggestions to improve the README. I really like them overall. However, please remove the "Project Highlights" and "App Preview" sections, and then I'll be happy with the changes. Once those updates are made, I will label the PR as _hacktoberfest-accepted_ and merge it.

**Comment by ayush-pandey047** [comment]
I have updated your readme.md you can check it.

**Comment by ayush-pandey047** [comment]
I HAVE CHANGE AS PER YOUR NEEDS.

**Comment by davidchocholaty** [comment]
Perfect, thank you! It looks good now, and the `hacktoberfest-accepted` label has already been set. The PR is ready to be merged. Thank you for your work.

**Review by davidchocholaty — APPROVED** [review]

**Pull request merged**